src/Orchard.Cms.Web/Modules/Orchard.OpenId/Controllers/AccessController.cs

```diff
@@ -267,15 +294,25 @@ public class AccessController : Controller
                 });
             }
 
+            var request = HttpContext.GetOpenIdConnectRequest();
```

Comment:
The `CheckConfiguration()` thingy won't work, as we need the `request` instance in the action (it was previously provided as an action parameter).

Concerning `ConfigurationError`, it seems a bit ugly, but if you believe it's better, I'm fine changing it.